src/libs/Notification/LocalNotification/BrowserNotifications.js

```diff
@@ -117,4 +117,15 @@ export default {
             onClick,
         });
     },
+
+    /**
+     * Create a notification to indicate that an update is available.
+     */
+    pushUpdateAvailableNotification() {
```

Comment:
Let's update this so that it takes a `version` parameter, then add an `onClick` handler that does:

```javascript
Onyx.merge(ONYXKEYS.UPDATE_VERSION, version);
```